cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -26,6 +26,10 @@ There are specific instructions for :ref:`upgrading existing CDAP installations
 
 Once installed, another page :ref:`covers configuration <configuration>` of the CDAP installation.
 
+Use these instructions if you aren't using a distribution such as :ref:`Cloudera <cloudera-configuring>`, 
+:ref:`Ambari <ambari-configuring>`, or the :ref:`standalone SDK <standalone-index>`. For the
```

Comment:
Use these instructions if you aren't using a cluster manager such as Cloudera Manager or Ambari. For the Mapr distribution, we have additional instructions to start with.  These instructions do not apply to the standalone SDK.  (or something like that)